HW0/P4/P4.py

```diff
@@ -30,47 +30,116 @@
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    x_coords, y_coords,z_coords, v_x, v_y, v_z = np.loadtxt('P4_trajectory.txt', 
+        delimiter=',',unpack=True)
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    ax.plot(x_coords, y_coords, z_coords,
+             '--b', label='True trajectory')
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
     #####################
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    x_coords, y_coords, z_coords = np.loadtxt('P4_measurements.txt', delimiter=',',unpack=True)
+    ax.plot(x_coords/rx, y_coords/ry, z_coords/rz,
+             '.g', label='Observed trajectory')
 
     #####################
     # Part 3:
     # Use the initial conditions and propagation matrix for prediction
     #####################
 
-    # A = ?
-    # a = ?
-    # s = ?
```

Comment:
Incredibly minor style point but I think we were supposed to uncomment the vars above and use those lines to initialize (aka what you did on the next few lines). Again, doesn't change anything, just may decrease the amount of unnecessary comments in the code.